src/client/util/http.cc

```diff
@@ -338,7 +340,9 @@ bool HTTP::ProcessI2PResponse() {
   std::uint16_t response_code = 0;
   m_Response >> http_version;
   m_Response >> response_code;
-  if (response_code == static_cast<std::uint16_t>(ResponseCode::HTTP_OK)) {
+  namespace http = boost::network::http;
```

Comment:
Since you use this twice, you may as well declare in namespace scope instead of local scope:

```diff
diff --git a/src/client/util/http.cc b/src/client/util/http.cc
index 070140b..35c2deb 100644
--- a/src/client/util/http.cc
+++ b/src/client/util/http.cc
@@ -52,6 +52,8 @@
 namespace kovri {
 namespace client {
 
+namespace http = boost::network::http;
+
 // TODO(unassigned): currently unused but will be useful
 // without needing to create a new object for each given URI
 bool HTTP::Download(
@@ -181,7 +183,6 @@ bool HTTP::DownloadViaClearnet() {
       Response response = client.get(request);
       LOG(trace) << "HTTP: Response " << kovri::core::LogNetMessageToString(response);
       // Test HTTP response status code
-      namespace http = boost::network::http;
       switch (response.status()) {
         // New download or cached version does not match, so re-download
         case http::basic_response<http::tags::http_server>::status_type::ok:
@@ -340,7 +341,6 @@ bool HTTP::ProcessI2PResponse() {
   std::uint16_t response_code = 0;
   m_Response >> http_version;
   m_Response >> response_code;
-  namespace http = boost::network::http;
   if (response_code ==
                http::basic_response<http::tags::http_server>::status_type::ok) {
     bool is_chunked = false;
```